client/src/components/layout/TopBar.jsx

```diff
@@ -19,7 +19,7 @@ export const TopBar = () => (
 
       <div className="menu__right">
         <li className="menu__item button">
-          <Link to="/squids">Squids</Link>
+          <Link to="/squids?pageNumber=1">Squids</Link>
```

Comment:
Curious if this change is necessary -- I think the Squids component should handle for no search params provided (defaulting to page 1)